packages/atomic/src/components/atomic-search-interface/atomic-aria-live.tsx

```diff
@@ -14,27 +13,64 @@ export interface UpdateLiveRegionEventArgs {
   shadow: false,
 })
 export class AtomicAriaLive {
-  @State() public regions: Record<string, string> = {};
+  @Element() private host!: HTMLAtomicAriaLiveElement;
+  @State() private message = '';
 
-  @Listen('atomic/accessibility/updateLiveRegion', {target: 'document'})
-  protected updateMessage({
-    detail: {message, region},
-  }: CustomEvent<UpdateLiveRegionEventArgs>) {
-    this.regions = {...this.regions, [region]: message};
+  /**
+   * Only the `atomic-aria-live` element with the greatest priority will be used to announce changes in the search interface.
+   *
+   * @internal
+   */
+  @Prop({reflect: true}) priority = 1;
+
+  private lastUpdatedRegion?: string;
+  private disconnectFindAriaLiveEvent?: () => void;
+
+  protected onFindAriaLive(args: FindAriaLiveEventArgs) {
+    if (args.element && args.element.priority > this.priority) {
+      return;
+    }
+    args.element = this.host;
+  }
+
+  @Method()
+  public async updateMessage(region: string, message: string) {
+    const wouldOverwriteAnotherRegion = region !== this.lastUpdatedRegion;
+    if (wouldOverwriteAnotherRegion) {
+      this.lastUpdatedRegion = region;
+      if (!message) {
+        return;
+      }
+    }
+    this.message = message;
+  }
+
+  public connectedCallback() {
```

Comment:
I spent an embarrassing amount of time trying to figure out a way to detect all the `aria-live` elements in the page and pick out the one with the highest priority. Unfortunately, what caused me the longest delay, was that Stencil's `@Listen` seems to somehow provide only a clone of the event's arguments to the method, which meant updating said arguments gave nothing to the event dispatcher.